### ISSUE TITLE
NTR: Arylsulfatase A high in blood or amniotic fluid (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8202]
New term request Arylsulfatase A high in blood or amniotic fluid (segal_080118211326):

**Comment by pnrobinson** [comment]
@MickeySegal
there is this
[Reduced leukocyte arylsulfatase A activity HP:0034863](https://hpo.jax.org/browse/term/HP:0034863)
I am not sure it is worthwhile to have separate terms, can we rename to "tissue activity"?
thoughts?

**Comment by MickeySegal** [comment]
The existing term "Reduced leukocyte arylsulfatase A activity" is good phrasing, but here we have INCREASED levels:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7555117/
In 1971, Wiesmann et al. reported a 90-fold increase in arylsulfatase A activity

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001112
- label: Elevated tissue arylsulfatase A activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: 
- def: Activity of arylsulfatase A (EC 3.1.6.8) above the upper limit of normal. The activity can be measured in multiple tissues including culutured fibroblasts and leukocytes or in the blood circulation.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8202
- synonyms:
